src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,114 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { PersonLink } from './PersonLink';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useParams } from 'react-router-dom';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  const { slug } = useParams();
+
+  const isSelected = people.find(person => person.slug === slug);
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setError('Something went wrong');
+      })
```

Comment:
The error handling in the `catch` block sets a generic error message 'Something went wrong'. It might be beneficial to log the actual error or provide more specific error messages to help with debugging and improve user feedback.